README.md

```diff
@@ -199,6 +199,92 @@ python benchmarks/benchmark_serving.py --tokenizer $tokenizer_path --num-prompts
 Please look at `deps/JetStream/benchmarks/README.md` for more information.
 
 
+
+## Run server with Ray Serve
+
+### Prerequisites
+
+If running on GKE:
+
+1. Follow instructions on [this link](https://github.com/GoogleCloudPlatform/ai-on-gke/tree/main/ray-on-gke/guides/tpu) to setup a GKE cluster and the TPU webhook.
+2. Follow instructions
+   [here](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/cloud-storage-fuse-csi-driver)
+   to enable GCSFuse for your cluster. This will be needed to store the
+   converted weights.
+3. Deploy one of the sample Kuberay cluster configurations:
+```bash
+kubectl apply -f kuberay/manifests/ray-cluster.tpu-v4-singlehost.yaml
+```
+or
+```bash
+kubectl apply -f kuberay/manifests/ray-cluster.tpu-v4-multihost.yaml
+```
+
+
+### Start a Ray Serve deployment
+
+Single-host (Llama2 7B):
+
+```bash
+export RAY_ADDRESS=http://localhost:8265
+
+kubectl port-forward svc/example-cluster-kuberay-head-svc 8265:8265 &
+
+ray job submit --runtime-env-json='{"working_dir": "."}' -- python run_ray_serve_interleave.py  --tpu_chips=4 --num_hosts=1 --size=7b --model_name=llama-2 --batch_size=32 --max_cache_length=2048 --tokenizer_path=/llama/tokenizer.model --checkpoint_path=/llama/ckpt --quantize_weights=True --quantize_type="int8_per_channel" --quantize_kv_cache=True --sharding_config="default_shardings/llama.yaml"
```

Comment:
I don't think `ray job submit` was intended to be used with Ray Serve in this way. If you run it like this, the Ray Serve application will be treated as a Ray job and not survive a restart. Could we provide an example that uses the `serve run` CLI or the KubeRay RayService? cc @ryanaoleary